application/apps/webassembly/matcher/spec/test.spec.ts

```diff
@@ -0,0 +1,239 @@
+let wasm: typeof import('../pkg');
+
+describe('test', function () {
+    beforeAll(async function () {
+        wasm = await import('../pkg');
+    });
+
+    function compareMulti(result: string, expected: { [key: string]: string }[]) {
+        const parsedResult: { [key: string]: string }[] = JSON.parse(result);
+        expected.forEach((e_map: { [key: string]: string }, i: number) => {
+            let iResult = parsedResult[i];
+            Object.entries(e_map).forEach((key_value) => {
+                expect(iResult[key_value[0]]).toBe(key_value[1]);
+            });
+        });
+    }
+
+    function matcher() {
+        const { Matcher } = wasm;
+        const matcher = Matcher.new();
+        const items: string = `
+        [
+            {
+                "name": "very_large_file.dlt",
+                "size": "20gb",
+                "path": "/home/user/Desktop/very_large_file.dlt"
+            },
+            {
+                "name": "medium_sized_file.txt",
+                "size": "15mb",
+                "path": "/home/user/Desktop/medium_sized_file.txt"
+            },
+            {
+                "name": "small_file.log",
+                "size": "630kb",
+                "path": "/home/user/Desktop/small_file.log"
+            }
+        ]`;
```

Comment:
we cannot use paths to local files in the scope of gitactions and we should not put it into repo (paths).
I think it would be nice to have it as config.
Take a look to: 
```
application/apps/rustcore/ts-bindings/spec/config.ts
application/apps/rustcore/ts-bindings/spec/defaults.json
application/apps/rustcore/ts-bindings/spec/common.ts
```
I believe you can at least copy/paste it into your solution.
The idea is:
- if test configuration isn't setup - we are using `default.json` as test configuration
- if test configuration is setup - we are using configuration with local files